contracts/sol6/IKyberDAO.sol

```diff
@@ -1,7 +1,9 @@
-pragma solidity 0.5.11;
+pragma solidity 0.6.6;
 
+import "./Dao/IEpochUtils.sol";
 
-interface IKyberDAO {
+
+interface IKyberDAO is IEpochUtils {
```

Comment:
so now can inherit interface.
its new to me.

Reply:
Yeah, didn't know too